From 12736401a399d878d3a6377c0ea8dd8ac48b2427 Mon Sep 17 00:00:00 2001
From: Jamin Lin <jamin_lin@aspeedtech.com>
Date: Mon, 28 Aug 2023 17:38:25 +0800
Subject: [PATCH] ast2600:dcscm: change kernel boot address

Signed-off-by: Jamin Lin <jamin_lin@aspeedtech.com>
---
 include/configs/evb_ast2600_spl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/evb_ast2600_spl.h b/include/configs/evb_ast2600_spl.h
index 23dd7ecd7f..d2357a1fd9 100644
--- a/include/configs/evb_ast2600_spl.h
+++ b/include/configs/evb_ast2600_spl.h
@@ -32,7 +32,7 @@
 #undef CONFIG_EXTRA_ENV_SETTINGS
 #define CONFIG_EXTRA_ENV_SETTINGS	\
 	"loadaddr=" STR(CONFIG_SYS_LOAD_ADDR) "\0"	\
-	"bootspi=fdt addr 20100000 && fdt header get fitsize totalsize && cp.b 20100000 ${loadaddr} ${fitsize} && bootm; echo Error loading kernel FIT image\0"	\
+	"bootspi=fdt addr 21520000 && fdt header get fitsize totalsize && cp.b 21520000 ${loadaddr} ${fitsize} && bootm; echo Error loading kernel FIT image\0"	\
 	"verify=yes\0"	\
 	""
 
-- 
2.25.1

